### PR TITLE
docs: acknowledgements

### PR DESCRIPTION
**Pull request by Oppen**
- **fix: add PR template, including acknowledgements reminder**
- **fix: acknowledge the giants we stand on**

### CODE REVIEW

**Review by MarcosNicolau — APPROVED** [review]
Great!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Optimization
+
+## Checklist
```
Comment (JulianVentura) [inline .github/pull_request_template.md:16]: Maybe we can add an item here to mark for functionality that could be improved or refactored later on. For instance, rewriting an algorithm to be easier to follow, or to improve its performance. Maybe that can lead to open a new issue which could be linked here as well.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Description of the pull request changes and motivation.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Optimization
```
Comment (uri-99) [inline .github/pull_request_template.md:13]:
```suggestion
- [ ] Optimization
- [ ] Refactor
```

**Review by uri-99 — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Checklist
+- [ ] Linked to Github Issue
+- [ ] This change depends on code or research by an external entity
+  - [ ] Acknowledgements were updated to give credit
+- [ ] Unit tests added
+- [ ] This change requires new documentation.
+  - [ ] Documentation has been added/updated.
+- [ ] This change is an Optimization
+  - [ ] Benchmarks added/run
```
Comment (uri-99) [inline .github/pull_request_template.md:24]:
```suggestion
  - [ ] Benchmarks added/run
- [ ] Has a known issue
  - [Link to the open issue addressing it]() 
```

Comment (uri-99) [reply .github/pull_request_template.md:24]: This suggestion is to address JulianVentura's comment

**Pull request merged**